util/csvresponse.py

```diff
@@ -91,6 +135,7 @@ def _create_files(self, base_path):
 
             with open(file_path, 'w') as filehandle:
                 self._create_csv(ds, filehandle)
+                filehandle.close()
```

Comment:
doesn't python close the filehandle when exiting the with statement?

Reply:
are you still going to remove the close?